src/Avalonia.Diagnostics/Diagnostics/ViewModels/MainViewModel.cs

```diff
@@ -90,10 +89,9 @@ public void ToggleFpsOverlay()
 
         public ConsoleViewModel Console { get; }
 
-        public ViewModelBase Content
```

Comment:
Technically Content and _content shouldn't be null, you initialize them in ctor. That's why I tried previously to add [MemberNotNull], but it required .net 6 sdk.

Reply:
But I don't think it's important as for devtools code